src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void setName(String name) {
+        super.setName(name);
+    }
+
+    @Override
+    public String getName() {
+        return super.getName();
+    }
```

Comment:
do we need to override these methods? they are by default available for all child classes